src/app/auth/features/user-manager-detail/user-manager-detail.component.ts

```diff
@@ -32,38 +57,167 @@ export class UserManagerDetailComponent implements OnInit {
 
   showTransition = false;
 
-  #route = inject(ActivatedRoute);
-  #router = inject(Router);
   #userService = inject(UserService);
 
-  #userId = computed(() => this.#route.snapshot.params['id']);
+  #location = inject(Location);
+
+  id = input.required<string>();
+  #user = this.#userService.user;
   #users = this.#userService.users;
 
-  /**
-   * Get the specific user based on the route parameter
-   */
-  user = computed(() => {
-    const userId = this.#userId();
-    const users = this.#users.value();
-    return users.find((user: DisplayUser) => user.id === userId);
+  #cachedUser = signal<DisplayUser | undefined>(undefined);
+
+  user = computed<DisplayUser | undefined>(() =>
+    this.#cachedUser() || !this.#user.error() ? this.#user.value() : undefined,
+  );
+
+  loading = computed<boolean>(
+    () => !this.#cachedUser() && this.#user.isLoading() && !this.#user.error(),
```

Comment:
```suggestion
    () => !this.#cachedUser() && this.#user.isLoading()
```

I don't think we should check for error here, this is just for the loading indicator which should not appear if there is an error.